gril/ril_constants.h

```diff
@@ -372,6 +372,17 @@
 #define RIL_REQUEST_VOICE_RADIO_TECH 108
 #define RIL_REQUEST_SET_INITIAL_ATTACH_APN 111
 
+#define RIL_REQUEST_IMS_REGISTRATION_STATE 112
+#define RIL_REQUEST_IMS_SEND_SMS 113
+#define RIL_REQUEST_GET_DATA_CALL_PROFILE 114
+#define RIL_REQUEST_SET_UICC_SUBSCRIPTION  115
+#define RIL_REQUEST_SET_DATA_SUBSCRIPTION  116
+#define RIL_REQUEST_SIM_TRANSMIT_BASIC 117
```

Comment:
Neither SET_DATA_SUBS or SIM_TRANSMIT_BASIC appear to be standard requests defined by any version of AOSP's ril.h header. If they are custom OEM requests, then they should be added in an OEM-specific header. Also, as it's not used in any of your new code, I would just leave them out of this change.